apps/web/modules/event-types/views/event-types-single-view.tsx

```diff
@@ -401,25 +402,24 @@ const EventTypePage = (props: EventTypeSetupProps & { allActiveWorkflows?: Workf
     formState: { isDirty: isFormDirty, dirtyFields },
   } = formMethods;
 
+  const assignedUsers = formMethods.watch("children");
+  const hosts = formMethods.watch("hosts");
+  const assignAllTeamMembers = formMethods.watch("assignAllTeamMembers");
+
```

Comment:
As we check values using the form watch method, the modal isn't shown if hosts or users are assigned, but the form isn't saved. Please fix this.

Reply:
ack

Reply:
I had thought since we are allowing user to navigate away from the form without saving, so this should be ok.
Anyway, updated to handle this scenario also.